core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -88,7 +88,7 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
   /** Default properties present in the currently defined defaults file. */
   lazy val defaultSparkProperties: HashMap[String, String] = {
     val defaultProperties = new HashMap[String, String]()
-    if (verbose) {
+    if (verbose && propertiesFile != null) {
```

Comment:
When there is a `spark-defaults.conf` file in the conf dir
A.Original logic:
- If `propertiesFile` is `not` specified, it will print:
  ```
  Using properties file: ... spark-defaults.conf
  ```
- If `propertiesFile` is specified, it will print:
  ```
  Using properties file: ... xxx.properties
  ```


B.Current logic:
- If `propertiesFile` is not specified, will print nothing
- If `propertiesFile` is specified, it will print:
  ```
  Using properties file: ... xxx.properties
  ```

It seems that the logic is `inconsistent` with the logic before this PR https://github.com/apache/spark/pull/46709

Reply:
The PR is updated